_moira.pyx

```diff
@@ -35,7 +35,7 @@ class MoiraException(Exception):
 __connected = False
 
 def _error(code):
-    raise MoiraException, (code, error_message(code))
+    raise MoiraException((code, error_message(code)))
```

Comment:
Are the double parens here correct?

Reply:
No, they weren't. I mistakenly thought it was passing a tuple of those 2 things.